src/page/categoryPage/CategoryPage.jsx

```diff
@@ -36,6 +36,10 @@ export default function CategoryPage() {
         <Navbar></Navbar>
       </section>
 
+      <section className="category-page-namespace page-sidebar">
```

Comment:
On line 12 and 25 there is commented code too remove it.

Reply:
Sure.